.github/workflows/publish_public_package.yaml

```diff
@@ -83,22 +84,26 @@ jobs:
             GalleryName = 'PSGallery'
             NuGetApiKey = $ApiKey
           }
-          # if ('true' -eq '${{ inputs.IsPrerelease }}')
-          # {
-          #   $Parameters.Add('PrereleaseTag', '${{ inputs.PrereleaseTag }}')
-          # }
-          # if (-Not [string]::IsNullOrEmpty('${{ inputs.OverrideModuleVersion }}'))
-          # {
-          #   $Parameters.Add('OverrideModuleVersion', '${{ inputs.OverrideModuleVersion }}')
-          # }
+          if ('true' -eq '${{ inputs.IsPrerelease }}')
+          {
+            Write-Host "Adding IsPrerelease"
+            Write-Host ${{ inputs.IsPrerelease }}
```

Comment:
I'm pretty sure our PowerShell linter enforces this, but as this PowerShell inside a YAML file, the linter didn't check it.

Per [AvoidUsingWriteHost](https://learn.microsoft.com/en-us/powershell/utility-modules/psscriptanalyzer/rules/avoidusingwritehost?view=ps-modules), "The use of Write-Host is greatly discouraged." It looks like these are basically just logging message, in which case `Write-Verbose` is what Microsoft would recommend we use.

Reply:
You are correct, Write-Host is discouraged.  Unfortunately, there are no alternatives that I am aware of for GitHub Action worksflows.  I've tested this all before, but I just tested it again.  Comments like this:

```
Write-Verbose "WV: Adding IsPrerelease"
Write-Verbose ${{ inputs.IsPrerelease }}
```

do not actually publish anything to the workflow console logs.  And a comment like this:

```
Write-Verbose "WV: Checking for prerelease with required version: " + $Version
```

actually throws an error and breaks the workflow. 

See here for example: https://github.com/cisagov/ScubaGear/actions/runs/9454334557/job/26041655409

 If you know of another `Write-<something>` that actually works, I would be happy to switch, but AFAICT there are no options.

Reply:
Ran a test on the various Write-*'s and found that host, output, and warning work.  Error throws an error.  Switch all hoss to output.  Also fixed a concatenation typo.